src/app/shared/palette/label/label-field.html

```diff
@@ -1 +1 @@
-<dl class="case-field"><dt><ccd-markdown [content]="caseField.label"></ccd-markdown></dt><dd></dd></dl>
+<dl class="case-field"><dt><ccd-markdown [content]="caseField.value || caseField.label"></ccd-markdown></dt><dd></dd></dl>
```

Comment:
aren't we breaking encapsulation of the caseField here? Ideally the caseField itself should know how to return it's label and where to get it from. 
Can we expose a method
`
getLabel() {
 return caseField.value || caseField.label
}
`
?

Reply:
Nope, that knowledge doesn't belong to the `CaseField`, it's specific to the `LabelField`. We certainly don't want other types of fields to start returning their values as a label.

Reply:
That's because this particular way of retrieving the label is specific only for the search view right?

Reply:
in theory then we could have a method getSearchResultLabel() on the CaseField

Reply:
even though I don't like it much either :)
But doing `[content]="caseField.value || caseField.label"` smells a bit to me :)

Reply:
I don't think leaking that logic anywhere outside of `LabelField`, which is the only one responsible for it, is good.
At best, I'd say we can introduce a `getLabel()` function inside `LabelField` to extract that from the template.